docs/content/en/contribute/general/guidelines_for_contributing.md/_index.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
the directory should not be named with `.md` in the directory name, that is the reason, why the build is failing